EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/attributeInput/DateAttributeInput.java

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2012-2017, Business Process Technology (BPT),
+ * http://bpt.hpi.uni-potsdam.de.
+ * All Rights Reserved.
+ *
+ *******************************************************************************/
+package de.hpi.unicorn.application.pages.input.generator.attributeInput;
+
+import de.hpi.unicorn.event.attribute.TypeTreeNode;
+import org.apache.log4j.Logger;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class DateAttributeInput extends AttributeInput {
+
+	private Date value;
+	private static final DateFormat dateFormatter = new SimpleDateFormat("yyyy/MM/dd'T'HH:mm");
+
+	static final Logger logger = Logger.getLogger(DateAttributeInput.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') "logger" is the name of a field in "AttributeInput". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2387)